notebooks/Analyze_Text.ipynb

```diff
@@ -143,7 +143,7 @@
     {
```

Comment:
<p>Seems like an improvement from previous results, probably from updated Watson?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>I think it's the same answer but in a different order. NLU isn't supposed to change results if you don't change the value of the <code>version</code>  argument (see the first code cell).</p>

Reply:
<p>On second, thought, looks like the confidence values have changed. Not sure what's going on. Maybe <code>&nbsp;version="2019-07-12"</code>  gives you a different actual version of the models now...</p>